tests/crm_test.py

```diff
@@ -1048,6 +1052,10 @@ def test_crm_show_summary(self):
         assert result.output == crm_show_summary
         result = runner.invoke(crm.cli, ['config', 'polling', 'interval', '30'], obj=db)
         print(sys.stderr, result.output)
+        result = runner.invoke(crm.cli, ['config', 'polling', 'interval', '30000'], obj=db)
+        print(sys.stderr, result.output)
+        assert result.exit_code == 2
+        assert crm_config_interval_too_big in result.output
         result = runner.invoke(crm.cli, ['show', 'summary'], obj=db)
```

Comment:
add separate test
1) set interval to valid value
2) check return code
3) set invalid interval
4) check rc and error message